docs/howto/use_package_storage_v2.md

```diff
@@ -16,6 +16,19 @@ We recommend to use proper versioning instead and follow these rules:
 * a package with `version < 1.0.0` is a technical preview.
 * a package with `version >= 1.0.0` can contain prerelease tags (beta1, SNAPSHOT, next) on its version to indicate its prerelease state.
 
+### Prerelease and stable version
+
+A flat storage structure has implications for exposing revisions in the Fleet Integrations UI.
+
+The Package Storage v1 allowed for pinning package revisions to particular stages and let package owners decide on when to promote the package.
+We received critical feedback about this approach as it required package owners to take a manual action to promote package revisions.
+The Package Storage v2 assumes that every published revision is exposed to Fleet users, and package owners can release their packages
+as snapshots or technical previews using semantic versioning (prerelease tags).
+
+Notice: if you prefer Fleet users not to pick up and install prereleases, it's recommended to be cautious with exposing new features
+until [kibana#122973](https://github.com/elastic/kibana/issues/122973) is implemented. With this feature, users can decide
+whether they can pick up prerelease package revisions.
```

Comment:
Maybe we can also recommend to focus more on local/internal development before publishing prerelease packages till https://github.com/elastic/kibana/issues/122973 is implemented.

Reply:
I added a sentence about the local dev environment.